android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -367,6 +367,23 @@ public void setInjectedJavaScript(WebView view, @Nullable String injectedJavaScr
   public void setMessagingEnabled(WebView view, boolean enabled) {
     ((RNCWebView) view).setMessagingEnabled(enabled);
   }
+   
+  @ReactProp(name = "incognito")
+  public void setIncognito(WebView view, boolean enabled) {
+    // Disable cookies
+    CookieManager.getInstance().setAcceptCookie(!enabled);
```

Comment:
Instead of not accepting cookies would it make more sense to use `removeAllCookies`? That way when you launch the WebView you get a completely new session as expected, but while the WebView is open, the website can still set cookies as needed. 

The reason I ask is because I pulled these changes that you have into my app, but the site that is used in my WebView needs to use cookies, so it won't work the way it is currently set up. However, I changed this line to be `CookieManager.getInstance().removeAllCookies(null);` and things started to work as expected.